src/modules/simulation/gz_bridge/gz_env.sh.in

```diff
@@ -1,8 +1,14 @@
 #!/usr/bin/env bash
-
 export PX4_GZ_MODELS=@PX4_SOURCE_DIR@/Tools/simulation/gz/models
 export PX4_GZ_WORLDS=@PX4_SOURCE_DIR@/Tools/simulation/gz/worlds
 export PX4_GZ_PLUGINS=@PX4_BINARY_DIR@/src/modules/simulation/gz_plugins
+export PX4_GZ_SERVER_CONFIG=@PX4_SOURCE_DIR@/src/modules/simulation/gz_bridge/server.config
 
 export GZ_SIM_RESOURCE_PATH=$GZ_SIM_RESOURCE_PATH:$PX4_GZ_MODELS:$PX4_GZ_WORLDS
 export GZ_SIM_SYSTEM_PLUGIN_PATH=$GZ_SIM_SYSTEM_PLUGIN_PATH:$PX4_GZ_PLUGINS
+export GZ_SIM_SERVER_CONFIG_PATH=$HOME/.gz/sim/px4/server.config
+
+# We load plugins specific for PX4 through our own server config file
+# See https://gazebosim.org/api/sim/8/server_config.html
+mkdir -p $HOME/.gz/sim/px4/
+cp $PX4_GZ_SERVER_CONFIG $GZ_SIM_SERVER_CONFIG_PATH
```

Comment:
Given everything else here is dependent on PX4 src & bin directory why not use the gz server config directly out of the PX4 src dir?

Separately we could also have an optional system install for this stuff, but if it makes sense I'd try to cleanly preserve the in tree build/run.

Reply:
You're right!